andela/reactors-cards-for-humanity
# Chore/143412447/integrate coveralls

#### What does this PR do?
 It involves integration of coverall and will update the  package.json scripts adding coveralls command, the travis.yml file by adding the _npm run coveralls_ command and the readme file by adding the badge.
 #### Description of Task to be completed?
Integrating Coveralls to the project repo and adding a coveralls badge.
 #### How should this be manually tested?
Run command _npm test_ to see if the coverage folder is created with the lvoc.info file that contains the tests analysis to be sent to coveralls.js by travis 
 #### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/143412447
